src/Symfony/Component/Notifier/Bridge/JoliNotif/README.md

```diff
@@ -0,0 +1,13 @@
+JoliNotif Notifier Bridge
+=========================
+
+Provides a [JoliNotif](https://github.com/jolicode/JoliNotif) integration for
+the Symfony Notifier Component.
+
+Resources
```

Comment:
This readme should have the description of its supported DSN for consistency with other component bridges where the readme documents the supported DSN.

Reply:
Done